examples/Benchmarking RDF against MDAnalysis.ipynb

```diff
@@ -12,7 +12,17 @@
    "cell_type": "code",
```

Comment:
<p>I'm not sure where this warning comes from, but I'd like to remove it before merging this PR. It appears in a few notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
This looks like a broken numpy installation.